### PR TITLE
Add man page for kpod-cp.1

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: docs/kpod-cp.1.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+% kpod(1) kpod-mount - Mount a working container's root filesystem.
```
Comment (TomSweeneyRedHat) [inline docs/kpod-cp.1.md]: should this be "kpod-cp - Copy "?

File: docs/kpod-cp.1.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+% Dan Walsh
+# kpod-cp "1" "August 2017" "kpod"
+
+## NAME
+kpod-cp - Copy files/folders between a container and the local filesystem.
+
+## Description
+We chose not to implement this feature even though the upstream docker project
```
Comment (TomSweeneyRedHat) [inline docs/kpod-cp.1.md]: suggest "feature even" to "feature in kpod"

File: docs/kpod-cp.1.md (hunk 1 above)
Comment (rhatdan) [inline docs/kpod-cp.1.md]: yup

File: docs/kpod-cp.1.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+has it. We have a much stronger capability.  Using standard kpod-mount and
```
Comment (TomSweeneyRedHat) [inline docs/kpod-cp.1.md]: Suggest changing "We have" to something like "The kpod tool has"

File: docs/kpod-cp.1.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+kpod-umount, we can take advantage of the entire linux command line, rather then
+just cp.
+
+If a user wants to copy contents out of a container or into a container, they
+can execute a few simple commands.
+
+You can copy from the container's file system to the local machine or the
+reverse, from the local filesystem to the container. 
+
+If you want to copy the /etc/foobar directory out of a container execute the
```
Comment (TomSweeneyRedHat) [inline docs/kpod-cp.1.md]: I'm hemming/hawing on this.  Maybe:  "container execute" to "container and onto the /tmp directory of the local host execute".

**Comment by rhatdan** [comment]
@mrunalp @runcom PTAL

This pull request is just for a man page, so it could be merged without waiting for the tests to complete.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

**Pull request merged**